### PR TITLE
Fix Storefinder library name

### LINKED ISSUE #10260: Fix Storefinder library name
We want to remove misc library and move store finder to @spartacus/store finder to make it in line with our general feature libraries approach.

### PR DESCRIPTION
**Pull request by dunqan**
Closes #10260

### CODE REVIEW

File: feature-libs/storefinder/README.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# Spartacus Store Finder Features Library
```
Comment (marlass) [inline feature-libs/storefinder/README.md]:
```suggestion
# Spartacus Store Finder Library
```

File: feature-libs/storefinder/README.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
...
+
+Spartacus Store Finder features can be added to the existing Spartacus application by running `ng add @spartacus/storefinder`. For more information about Spartacus schematics, visit the [official Spartacus schematics documentation page](https://sap.github.io/spartacus-docs/schematics/).
```
Comment (marlass) [inline feature-libs/storefinder/README.md]:
```suggestion
Spartacus Store Finder can be added to the existing Spartacus application by running `ng add @spartacus/storefinder`. For more information about Spartacus schematics, visit the [official Spartacus schematics documentation page](https://sap.github.io/spartacus-docs/schematics/).
```

File: feature-libs/storefinder/schematics/add-storefinder/schema.json [hunk 3]
```diff
@@ -13,7 +13,7 @@
     },
     "lazy": {
       "type": "boolean",
-      "description": "Lazy load the misc features.",
+      "description": "Lazy load the storefinder features.",
       "default": true
     },
     "features": {
```
Comment (znikola) [inline feature-libs/storefinder/schematics/add-storefinder/schema.json]: We don't need the interactive prompt now.

File: projects/schematics/src/migrations/3_0/storefinder/storefinder_spec.ts [hunk 4]
```diff
@@ -46,7 +46,7 @@ import { StoreFinderModule } from '@spartacus/misc/storefinder';
     B2cStorefrontModule.withConfig({}),
     StoreFinderRootModule,
     StoreFinderModule,
-    
+
```
Comment (znikola) [inline projects/schematics/src/migrations/3_0/storefinder/storefinder_spec.ts]: Do we need this empty line? :)

**Review by marlass — CHANGES_REQUESTED** [review]
We need to remove interactive form for storefinder schematics with feature select

**Review by znikola — DISMISSED** [review]

File: feature-libs/storefinder/schematics/add-storefinder/schema.json (hunk 3 above)
Comment (dunqan) [reply feature-libs/storefinder/schematics/add-storefinder/schema.json]: Agreed

File: projects/schematics/src/migrations/3_0/storefinder/storefinder_spec.ts (hunk 4 above)
Comment (dunqan) [reply projects/schematics/src/migrations/3_0/storefinder/storefinder_spec.ts]: yes, but it has to be empty with spaces unfortunately...

**Review by marlass — APPROVED** [review]

**Pull request merged**